e3db/types/search_params.py

```diff
@@ -2,7 +2,46 @@
 
 class Params(object):
     def __init__(self, condition="OR", strategy="EXACT", keys=None, values=None, writer_ids=None, user_ids=None, record_ids=None, content_types=None, plain={}):
-        # python3 is picky about uuids, cant pass a uuid to uuid.UUID
+        """
+        Initialize the Search Params class.
+
+        This object type is used to build the params for searching with the search service.
```

Comment:
Same as other comment, remove Search Service in favor of either "server" or "E3DB"